model_compression_toolkit/quantizers_infrastructure/pytorch/quantizer_utils.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import Tuple
+
+import torch
+import numpy as np
+
+
+def get_working_device():
+    """
+    Get the working device of the environment
+
+    Returns:
+        Device "cuda" if GPU is available, else "cpu"
+
+    """
+    return torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Just verifying - this method is duplicated (from 'core/pytorch/utils') because we don't want to import from MCT into the infrastructure?

Also, consider using constants for "cuda" and "cpu".

Reply:
I see now that this entire file is duplicated so I'm guessing that the answer to my question is positive.
Leaving this comment anyway just to make sure.

Reply:
you are correct. i'll use constants